gpcontrib/arenadata_toolkit/src/dbsize.c

```diff
@@ -35,11 +32,11 @@ PG_MODULE_MAGIC;
 
 static int64 calculate_relation_size(Relation rel, ForkNumber forknum);
 static int64 get_heap_storage_total_bytes(Relation rel,
-							 ForkNumber forknum, char *relpath);
+										  ForkNumber forknum, char *relpath);
```

Comment:
In dbsize.c not relevant code style changes are a subject for a separate PR as any refactoring.